<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnScale">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>598</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMove">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>159</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Перенос</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Импортировать</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtDeltaY"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="txtDeltaX"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtDeltaZ"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Смещения по осям</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>238</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="1" column="0">
      <widget class="QLineEdit" name="txtRotCenterX"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtRotCenterY"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtRotCenterZ"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Центр поворота</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Углы поворота вокруг трех осей</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="txtRotAngleOX"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtRotAngleOY"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="txtRotAngleOZ"/>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>631</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>448</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="1" column="0">
      <widget class="QLineEdit" name="txtScaleCenterX"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="txtScaleZ"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtScaleCenterZ"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtScaleY"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Коеффиценты масштабирования</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Центр масштаирования</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="txtScaleX"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtScaleCenterY"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
